src/components/publishing/header/feature_header.tsx

```diff
@@ -0,0 +1,261 @@
+import React from "react"
+import sizeMe from "react-sizeme"
+import styled from "styled-components"
+import { pMedia } from "../../helpers"
+import Fonts from "../fonts"
+import AuthorDate from "./author_date"
+
+function renderFeatureAsset(url, layout, width) {
+  if (layout === "fullscreen") {
+    return (
+      <div>
+        {renderAsset(url)}
+        <Overlay />
+      </div>
+    )
+  } else if (layout === "split" && width > 600) {
+    return renderAsset(url)
+  } else {
+    return false
+  }
+}
+
+function renderMobileSplitAsset(url, layout, width) {
+  if (layout === "split" && width < 600) {
+    return renderAsset(url)
+  } else {
+    return false
+  }
+}
+
+function renderAsset(url) {
+  if (isVideo(url)) {
+    return (
+      <FeatureVideoContainer>
+        <FeatureVideo src={url} autoPlay controls={false} loop muted playsInline />
+      </FeatureVideoContainer>
+    )
+  } else {
+    return <FeatureImage src={url} />
+  }
+}
+
+function renderTextLayoutAsset(url, layout) {
+  if (layout === "text") {
+    if (isVideo(url)) {
+      return <Video src={url} autoPlay controls={false} loop muted playsInline />
+    } else {
+      return <Image src={url} />
+    }
+  } else {
+    return false
+  }
+}
+
+function isVideo(url) {
+  return url.includes("mp4")
+}
+
+interface FeatureHeaderProps {
+  article?: any
+  size?: any
+}
+
+const FeatureHeader: React.SFC<FeatureHeaderProps> = props => {
+  const { article } = props
+  const { width } = props.size
+  const hero = article.hero_section
+  return (
```

Comment:
Realized when starting to implement image-set in Positron that passing larger objects doesn't account for Backbone's attributes.  Hadn't considered that we will need to convert our articles to json before passing in. (this is maybe more of a problem for positron than force)